libs/core/src/lib/nested-list/nested-list-keyboard.service.ts

```diff
@@ -56,7 +55,7 @@ export class NestedListKeyboardService {
     private getAllListItems(list: NestedListInterface): NestedItemInterface[] {
 
         const _items: NestedItemInterface[] = [];
-        if (list && list.nestedItems) {
+        if (list && list.nestedItems && list.nestedItems.toArray()) {
```

Comment:
I `nestedItems` is a `QueryList`. You can use `QueryList.length` to improve performance a bit.